plans/example/output.go

```diff
@@ -1,14 +1,20 @@
 package main
 
 import (
+	"os"
+
 	"github.com/testground/sdk-go/run"
 	"github.com/testground/sdk-go/runtime"
 )
 
 // Demonstrate test output functions
 // This method emits two Messages and one Metric
 func ExampleOutput(runenv *runtime.RunEnv, initCtx *run.InitContext) error {
-	runenv.RecordMessage("Hello, World.")
+	if msg := os.Getenv("TESTGROUND_HELLO_MESSAGE"); msg != "" {
+		runenv.RecordMessage(msg)
+	} else {
+		runenv.RecordMessage("Hello, World.")
+	}
```

Comment:
That's an example but not a test: Remember we're building TESTground, we want our codebase to be tested thoroughly, if we break this feature in 3 months, this example won't detect it.

Here is an example of testing additional hosts:
https://github.com/testground/testground/blob/master/integration_tests/20_docker_additional_hosts.sh
https://github.com/testground/testground/blob/master/plans/additional_hosts/main.go

Note how the code fails if the host is not set.

Another example with the mixed builders feature:
https://github.com/testground/testground/blob/master/integration_tests/15_docker_mixed_builders_configuration.sh
https://github.com/testground/testground/blob/9c13f3bbe956c3cc29591db4b8b3c63b6fa1f6dc/plans/_integrations_mixed_builders/go/main.go#L11

Note how the test will fail if the values are not set correctly.

Reply:
(The scripts are registered in the Makefile for now: https://github.com/testground/testground/blob/9c13f3bbe956c3cc29591db4b8b3c63b6fa1f6dc/Makefile#L61)

Reply:
Ok going to see to add `21_docker_additional_envs.sh` then and the like :)

Reply:
Added the additional test and also enabled the `20_*.sh` test, as that one wasn't enabled yet.